lib/engine/game/g_1822_pnw/game.rb

```diff
@@ -768,6 +762,10 @@ def unassociated_minors
           @corporations.select { |c| c.floated? && c.type == :minor && !MINOR_ASSOCIATIONS.include?(c.id) }
         end
 
+        def regionals
+          @corporations.select { |c| regional_railway?(c) }
```

Comment:
Cache this?

Reply:
@corporations can change, so didn't cache it.  Added comment indicating that.